data/json/furniture.json

```diff
@@ -2091,7 +2091,7 @@
         "type" : "furniture",
         "id" : "f_still",
         "name": "still",
-        "description": "Invaluable equipment in brewery and chemisty, that distills liquids.",
+        "description": "Invaluable equipment in brewery and chemistry, that distills liquids.",
```

Comment:
This sentence is kinda wonky. Anyone have a better one?

Reply:
`An essential and critical component for brewing and chemistry that is capable of elaborate distillation processes.`
`A fundamental apparatus used in brewery and chemistry that allows for refining liquid mixtures.`